Dockerfile

```diff
@@ -85,6 +85,8 @@ RUN \
       python3.10 python3.10-dev python3.10-distutils \
       dpkg-dev \
       cmake \
+      libjson-perl \
+      libdigest-crc-perl \
```

Comment:
So, I think choosing two languages to show the benefit of using KSY is great, and apologies for calling into question the validity of this work but...

I'm not sure about the choice of Perl here, do we have customers that want to parse SBP with Perl?  What's the motivation for supporting it?  Picking something with customer motivation (e.g. C#) would make more sense in my opinion.

Again, I don't want to torpedo this work because I don't agree with the choice of Perl, but if the DevInfra team is going to support this, we should understand the motivations.

Reply:
I did have a go at implementing a version for C#, but I quickly discovered that there are some issues with the .cs files generated by kaitai-struct-compiler.  I played around with it for a bit but then gave up and moved onto another language which I am more familiar with (I also ended up finding some minor issues in the generated perl code, but I managed to work around those).

Ideally, one of the following will happen in the future:
1. The bugs will get fixed in kaitai-struct-compiler
2. Somebody with sufficient motivation implements fixups for the generated csharp output

But my main motivation was to have working bindings for at least 2 languages (since this forced me to do some substantial refactoring which will make it easier to add even more languages in the future)